src/components/HeatMap/HeatMap.js

```diff
@@ -1,16 +1,71 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import table from './table.png';
-import config from '../../config';
-import { Wrapper, Image } from './HeatMap.styles';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Row from './Row';
+import Header from './Header';
+import Footer from './Footer';
+import { Wrapper } from './HeatMap.styles';
 
 
-export default function HeatMap() {
+const dayLabels = [
+  'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday',
+];
+
+export default function HeatMap({ data }) {
+  const [tableData, setTableData] = useState([]);
+  const [activeCell, setActiveCell] = useState([]);
+  const [timezone, setTimezone] = useState('');
+  const [timezoneDifference, setTimezoneDifference] = useState(0);
+
+
+  useEffect(() => {
+    const clientTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    const clientTimezoneDifference = (new Date().getTimezoneOffset()) * 60;
```

Comment:
I'm not sure what you use the timezone difference for. I didn't use it in my implementation. You're heatmap also seems to have an offset of two hours compared to mine. On top is yours, below mine
![image](https://user-images.githubusercontent.com/9722792/81541330-39e43280-9373-11ea-8ddd-e58eccbec37a.png)
![image](https://user-images.githubusercontent.com/9722792/81541347-3f417d00-9373-11ea-8ddc-9dea32a1e36d.png)

Reply:
But also here a state doesn't seem necessary. A simple function to get the timezone and it's offset should be enough

Reply:
So I take the  created_utc and convert to the user/visitor timezone, since the created_utc refers to the hour the autor of the subreddit created the post. I used a plugin to change the timezone of my browser to see how different it woul look like when visiting the site from Berlin or from Honolulu (for example).

Reply:
It sounded really logic at some point ... but now I am not so sure